training/distributed_training/pytorch/model_parallel/gpt2/fp16/__init__.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This file is copyright NVIDA - What is the original source of this file and how has it been modified? (Applies to at least 5 files in this PR)